src/ResultWriter/PostProcessor.cpp

```diff
@@ -0,0 +1,46 @@
+#include "PostProcessor.h"
```

Comment:
Add the common header to all new files. You can copy/paste it from one of the existing files and replace the author with your name.